### ISSUE TITLE
Extra comma in Collection list

### ISSUE THREAD

**Issue by aetaylor** [MehdiK/Humanizer#390]
There appears to be a slight bug in the humanised collections which includes an extra comma:

    One, Two, Three, and Four

the comma after Three should not be there.

**Comment by darrellnorton** [comment]
It's actually ok to be there. It's called an Oxford comma or serial comma (http://en.wikipedia.org/wiki/Serial_comma). Whether it is included or not depends on a couple factors described in the Wikipedia article. 

Since there is no universal decision I do not think it is a bug.

**Comment by justin-edwards** [comment]
I will admit that I took a [slightly opinionated](https://github.com/MehdiK/Humanizer/blob/master/src/Humanizer.Tests/CollectionHumanizeTests.cs#L52-L63) stance in regards to the Oxford comma since, in my experience, it more often reduces ambiguity than introduces it. It could be made optional(and that probably wouldn't take much work after #386), but like I said I was feeling a little opinionated that day.

**Comment by hazzik** [comment]
![fychx](https://cloud.githubusercontent.com/assets/144791/6478591/6068d496-c29b-11e4-961a-0f25ca50ff71.jpg)

**Comment by MehdiK** [comment]
Thanks for the discussion guys. Closing this thread now.

**Comment by hazzik** [comment]
Actually we need to do a research on usage of serial comma by English speaking country and implement this feature as regional differences: en-US, en-GB, en-AU, en-NZ, etc

**Comment by aetaylor** [comment]
Hazzik is correct. The unfortunately named 'Oxford' comma is frequently used by Americans but is generally regarded as a gramatical error in British English, although it has it's uses in certain edge-case scenarios. (see: http://www.grammar-monster.com/lessons/commas_in_lists.htm)
Since no-one is going to be truely happy about this it may be better to include an option flag, otherwise us Brits are going to have to continue to re-parse Humanizer's output in order to correct it ;o)

**Comment by hazzik** [comment]
I think we need to be able to specify conjunction word, something like this:

```
new { 1, 2, 3, 4, 5}.Humanize(", and");
new { 1, 2, 3, 4, 5}.Humanize(", or");
new { 1, 2, 3, 4, 5}.Humanize(", и");
```

**Comment by MehdiK** [comment]
Good point @hazzik. Agreed. We could also provide a configuration point for the separator or CollectionFormatter. If the config and the separator on the method call are both missing, we could then fall back to a collection formatter configured per locale.

**Comment by joelblake1** [comment]
Hi, 

I've just encountered this issue and as a Brit I can confirm the 'oxford comma' is not used in Great Britain. 

Is there any resolution to this issue in the pipeline? I'm reticent to introduce a home-grown workaround if there's a fix on the horizon...

Thanks

Joel

**Comment by onovotny** [comment]
Any updates on this? Is this still a general to-do? Should this be in for 2.0 or vNext?

**Comment by hazzik** [comment]
vNext

**Comment by clairernovotny** [comment]
Thanks for reporting this and for the clear examples, @aetaylor. This is now fixed on `main` via #1857: `en-GB` collection lists use British regular-list punctuation (for example, `one, two and three`), while neutral `en`, `en-US`, and `en-IN` preserve their existing Oxford-comma behavior.
